### PR TITLE
Fix the array handling

### PR DESCRIPTION
**Pull request by michaloo**
- support array changing

### CODE REVIEW

File: server/compute.js [hunk 1]
```diff
@@ -143,6 +143,11 @@ module.exports = function compute({ changes = {}, user, segments, events = [] },
     if (d.kind === "N" || d.kind === "E") {
       _.set(memo, d.path, d.rhs);
     }
+    // when we have an array updated we set the whole
+    // array in `changed` constant
+    if (d.kind === "A") {
+      _.set(memo, d.path, _.get(updatedUser, d.path, []));
```
Comment (sbellity) [inline server/compute.js]: @michaloo what if we have this : 

- previous value is string -> new value is array
- previous value is array -> new value is string

are those cases handled ?

Comment (sbellity) [inline server/compute.js]: can you please add tests for those ?

Comment (michaloo) [inline server/compute.js]: Yes, I'm seeing that is a deeper case, I also thought about tests here, really scary to modify it without testing.

**Comment by michaloo** [comment]
[hull-processor](https://trello.com/c/A9HgWEkZ/182-hull-processor)

**Pull request merged**